src/wazuh_modules/syscollector/tests/sysCollectorImp/syscollectorImp_test.cpp

```diff
@@ -2327,3 +2327,123 @@ TEST_F(SyscollectorImpTest, PackagesDuplicated)
         t.join();
     }
 }
+
+TEST_F(SyscollectorImpTest, sanitizeJsonValues)
+{
+    const auto spInfoWrapper{std::make_shared<SysInfoWrapper>()};
+    EXPECT_CALL(*spInfoWrapper, os()).WillRepeatedly(Return(nlohmann::json::parse(
```

Comment:
it may be useful to include an example that has a double space between two words, that case should not be altered.
`Example “Microsoft    Windows    7”`